### ISSUE TITLE
Question about newer Debian versions

### ISSUE THREAD

**Issue by n3gwg** [AllStarLink/ASL-Live-Build#49]
Developers, et alia:

I am going to be experimenting with All Star for the first time in the next month.

Downloading the 2.00 Beta version it seems to run on Debian 10 (Buster) and I am wondering if the current source code can be run on Debian 11 (Bullseye) or Debian 12 (Bookworm)?

I am also curious when a new image (iso) might be published that is based on Bullseye or even better Bookworm?

Thank you for all your hard work.

73,
Stuart, N3GWG

**Comment by tsawyer** [comment]
For now we're recommending https://community.allstarlink.org/t/awia-another-way-to-install-asl2-beta/20711 to install the ASL2 beta.

**Comment by n3gwg** [comment]
Sir,

If I installed All Star using the All Star Link Beta-2.0.0 ISO, would it work to just upgrade Debian to the latest version?

Stuart

**Comment by tsawyer** [comment]
I'm not sure. Install a Debian net install iso is sure to work. https://www.debian.org/CD/netinst/

When prompted, don't install a desktop, do install a ssh server.

**Comment by wd5m** [comment]
I have upgraded the RPI Beta image to Deiban 11. It's time consuming, but
the method is the same as upgrading from Debian 11 to 12.
I used the following step by step instructions for upgrading to Debian 12
(bookworm),
EXCEPT I replaced "buster" with "bullseye" in the step that modifies
"/etc/apt/sources.list".
https://www.makeuseof.com/upgrade-to-debian-12-bookworm-from-debian-11/

I will be trying the method to compile the beta source in the future.

David McAnally
WD5M
